css/styles.css

```diff
@@ -1,850 +1,1456 @@
+    height: 944px;
+    padding: 0;
+    margin: 0 auto;
+  }
+
+  .form {
+    width: 375px;
+  }
+
+  .cont-para {
+    width: 327px;
+    height: 216px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    text-align: center;
+    color: #172b4d;
+    margin: 0 auto;
+    margin-top: 5rem;
+  }
+
+  .form-fieldset {
+    width: 375px;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    gap: 2rem;
+    border: none;
+  }
+
+  .form-text {
+    width: 375px;
+    height: 48px;
+    margin-left: 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    border: 0;
+    padding: 0;
+    border-bottom: 1px solid #888;
+  }
+
+  .form-msg {
+    height: 114px;
+    background: #fbf8f7;
+    width: 375px;
+  }
+
+  #form-submit {
+    width: 131px;
+    height: 40px;
+    background-color: #ff6b00;
+    margin: auto;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+    padding-left: 15px;
+  }
+
+  #form-submit:hover {
+    background-color: #fff5e1;
+    border: 1px solid #f55800;
+    color: #f55800;
+  }
+
+  #form-submit:active {
+    background-color: #e05304;
+    color: #f4f1f0;
+  }
+
+  #form-submit:disabled {
+    background-color: #f4f1f0;
+    color: #5b5957;
+  }
+
+  .form-icon-container {
+    justify-content: center;
+    align-items: center;
+    padding: 10px;
+  }
+
+  .form-icon-element {
+    background-color: #fff;
+  }
+
+  hr {
+    border-top: 1px solid #888;
+    width: 100%;
+    margin-bottom: 0;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .header-container {
+    display: flex;
+    background: #3e3c3c;
+    align-items: center;
+    justify-content: center;
+    gap: 1.5%;
+  }
+
+  .header-item4 {
+    display: none;
+  }
+
+  .header-item3 {
+    width: 50%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    color: #fff;
+    text-decoration: none;
+    padding-left: 5%;
+  }
+
+  .header-item5,
+  .header-item6,
+  .header-item7,
+  .header-item8 {
+    width: 7%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .intro-section-container {
+    width: 100%;
+    height: 870px;
+    background: #1c1a19;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    background-image: url(../src/bg2.png);
+    background-repeat: no-repeat;
+    background-size: 40% 100%;
+    background-position: right;
+  }
+
```

Comment:
- [ ] **Good job so far!**  👍 👍 
Remember it is important that you try to replicate the [Figma design](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278&scaling=min-zoom&page-id=1%3A1471&hide-ui=1) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

    - It would be better to fix the padding of this section to match this 👇  of Figma design,
    
    ![image](https://user-images.githubusercontent.com/102757126/199932774-758cc00a-a5f2-4043-954d-d1bfb00a1309.png)

    **Your design**
    ![image](https://user-images.githubusercontent.com/102757126/199935304-7ae712b9-6f97-4c15-9b89-6dcd1f229310.png)